src/scripts/main.js

```
@@ -354,7 +354,35 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const table = document.querySelector('.dashboard');
```

Ensure that the '.dashboard' element exists in the DOM before attempting to append rows to it. If it doesn't exist, the script will fail silently. Consider adding a check to handle this scenario gracefully.